### PR TITLE
Add package-composite-v5 tag to SqlVulnerabilityAssessment API

### PR DESCRIPTION
**Pull request by shimonar1171**
### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [X] Other, Update API tag 
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [X] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [X] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [X] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [X] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following appy to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [X] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/sql/resource-manager/readme.md [hunk 1]
```diff
@@ -312,6 +312,12 @@ input-file:
 - ./Microsoft.Sql/preview/2020-11-01-preview/DatabaseRecommendedActions.json
 - ./Microsoft.Sql/preview/2020-11-01-preview/DatabaseSchemas.json
 - ./Microsoft.Sql/preview/2020-11-01-preview/DatabaseSecurityAlertPolicies.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/DatabaseSqlVulnerabilityAssessmentBaselines.json
```
Comment (ericshape) [inline specification/sql/resource-manager/readme.md:315]: Please add the new JSON files by alphabetical ordering.

Comment (ericshape) [reply specification/sql/resource-manager/readme.md:315]: please address this comment

File: specification/sql/resource-manager/readme.md [hunk 2]
```diff
@@ -37,137 +37,137 @@ These settings apply only when `--tag=package-preview-2022-02` is specified on t
...
+- ./Microsoft.Sql/preview/2022-02-01-preview/SyncGroups.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/SyncMembers.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/TdeCertificates.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/TimeZones.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/TransparentDataEncryptions.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/Usages.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/VirtualClusters.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/VirtualNetworkRules.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/WorkloadClassifiers.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/WorkloadGroups.json
```
Comment (ericshape) [inline specification/sql/resource-manager/readme.md:170]: if we did not change anything in 2022-02 tag, please revert this format chagne.

**Review by ericshape — APPROVED** [review]
LGTM

**Comment by xiaoxuqi-ms** [comment]
Hi @tadelesh , could you help to take a look at the SDK generation error for Go?

**Comment by tadelesh** [comment]
> Hi @tadelesh , could you help to take a look at the SDK generation error for Go?

Fixed.

**Comment by tadelesh** [comment]
> Hi @tadelesh , could you help to take a look at the SDK generation error for Go?

Fixed

**Comment by ericshape** [comment]
> Hi @shimonar1171, one or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow [breaking change policy](https://aka.ms/AzBreakingChangesPolicy) to request breaking change review and approval before proceeding swagger PR review. **Action**: To initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes). If you want to know the production traffic statistic, please see [ARM Traffic statistic](https://portal.azure-devex-tools.com/api/pr/query-traffic?pr=Azure/azure-rest-api-specs/19953&days=5). If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ms/swaggerfeedback. **Note**: To avoid breaking change, you can refer to [Shift Left Solution](https://aka.ms/shiftleft) for detecting breaking change in early phase at your service code repository.

please ask for breaking change review

**Comment by fengzhou-msft** [comment]
For azure-sdk-for-net-track2 failure, the pipeline treats this RP as a new SDK in the directory under `sdk/sql`, but there's existing sql SDK under `sdk/sqlmanagement`. Please ignore the failure for this PR. 

@chunyu3, is there any configuration to fix the swagger to SDK directory mapping?

**Review by xiaoxuqi-ms — APPROVED** [review]

**Comment by ericshape** [comment]
@xiaoxuqi-ms do we know why we still cannot merge this PR? This PR is fixing https://github.com/Azure/azure-rest-api-specs/issues/20067

**Comment by ericshape** [comment]
@tadelesh if we plan to fix the golang package issue in SQL RP, can we create another PR instead of testing on this PR?  
This PR is fixing https://github.com/Azure/azure-rest-api-specs/issues/20067  Thanks!

**Comment by tadelesh** [comment]
> @tadelesh if we plan to fix the golang package issue in SQL RP, can we create another PR instead of testing on this PR? This PR is fixing #20067 Thanks!

Sorry for the wrong commit. As the fix will cause Go SDK naming collision and breaking changes which is not acceptable, I need to fix it within this PR. But I'll test it locally first.

**Comment by ericshape** [comment]
> > @tadelesh if we plan to fix the golang package issue in SQL RP, can we create another PR instead of testing on this PR? This PR is fixing #20067 Thanks!
> 
> @ericshape Sorry for the wrong commit. As the fix will cause Go SDK naming collision and breaking changes which is not acceptable, I need to fix it within this PR. But I'll test it locally first.

Thanks! Do we know when we could merge this PR? The golang package commits have lasted almost two weeks.

**Comment by tadelesh** [comment]
> > > @tadelesh if we plan to fix the golang package issue in SQL RP, can we create another PR instead of testing on this PR? This PR is fixing #20067 Thanks!
> > 
> > 
> > @ericshape Sorry for the wrong commit. As the fix will cause Go SDK naming collision and breaking changes which is not acceptable, I need to fix it within this PR. But I'll test it locally first.
> 
> Thanks! Do we know when we could merge this PR? The golang package commits have lasted almost two weeks.

I've resolved the naming collision for Go. The last question is about the breaking changes. It seems the breaking is introduced by this [PR](https://github.com/Azure/azure-rest-api-specs/pull/19820). I'm asking swagger tooling team to check why there is no breaking change review request. Do you know any context about it?

**Comment by tadelesh** [comment]
Approve for Go breaking as it is not introduced by this PR.

**Pull request merged**